misc_scripts/change_toolchain_version.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python3
+"""
+change_toolchain_version: change the toolchain version in the GitHub Actions
+workflow configurations.
+"""
+
+# isort: STDLIB
+import argparse
+
+KEY_LSRT = r"# LOWEST SUPPORTED RUST TOOLCHAIN"
+KEY_CDRT = r"# CURRENT DEVELOPMENT RUST TOOLCHAIN"
+
+
+def main():
+    """
+    Main method
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("toolchain", choices=["lowest", "current"])
+    parser.add_argument("file")
+    parser.add_argument("old_version")
+    parser.add_argument("new_version")
+    args = parser.parse_args()
+
+    filename = args.file
+
+    if args.toolchain == "lowest":
+        search_key = KEY_LSRT
+    elif args.toolchain == "current":
+        search_key = KEY_CDRT
+
+    old_verstring = args.old_version + r"  " + search_key
+    new_verstring = args.new_version + r"  " + search_key
+
+    print("Old version: %s" % args.old_version)
```

Comment:
Now that the script is more mature, this is a good time to drop the debugging output.

Reply:
Resolved.